### PR TITLE
COMMERCE-11553 - Product with variants shows Unavailable label even though it has enough stock

### PR DESCRIPTION
**Pull request by victorcdp**
Related Issue: [COMMERCE-11553](https://issues.liferay.com/browse/COMMERCE-11553)

### CODE REVIEW

**Comment by victorcdp** [comment]
ci:test:sf

**Comment by victorcdp** [comment]
ci:test:relevant

**Comment by victorcdp** [comment]
ci:test:commerce

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/renderer/AvailabilityLabelInfoItemRenderer.java [hunk 1]
```diff
@@ -143,6 +143,13 @@ public void render(
 							"default"));
 				}
 			}
+			else {
+				httpServletRequest.setAttribute(
+					"liferay-commerce:availability-label:label",
+					StringPool.BLANK);
+				httpServletRequest.setAttribute(
+					"liferay-commerce:availability-label:labelType", "default");
+			}
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/renderer/AvailabilityLabelInfoItemRenderer.java:152]: I think it's better to move this before the if and clear these values anytime there's a value coming from another request

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/renderer/AvailabilityLabelInfoItemRenderer.java (hunk 1 above)
Comment (victorcdp) [reply modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/renderer/AvailabilityLabelInfoItemRenderer.java]: @andrea-ale-sbarra can you see if it's alright now? Thank you.

**Comment by andrea-ale-sbarra** [comment]
resent here https://github.com/liferay-commerce/liferay-portal/pull/3828

**Pull request closed without merging**